staff/templates/staff/reports/orga-report.tex

```diff
@@ -3,7 +3,10 @@
 \documentclass{article}
 \usepackage{orgacerts}
 
-% Pfad zum Ophasen-Wesen
+% Pfad zu Bildern
+\newcommand{\logoFS}{logo_fs.png}
```

Comment:
Most of the logos are already presend in the website app as svg. I think it is better to use the svg versions and to store them only once